### ISSUE TITLE
Louvolite motorised blinds support

### ISSUE THREAD

**Issue by navoshta** [dubocr/homebridge-tahoma#287]
I got motorised roller blinds installed recently; I initially got a quote for Somfy, but the blinds company must have changed their vendor, as they ended up installing Louvolite blinds instead (bummer!). Unfortunately, there are no existing Homebridge plugins that work with Louvolite right now, as far as I'm aware at least. I know it's a long shot, but as this plugin supports multiple platforms (Somfy, Atlantic, Sauter, etc.), is there any chance for Louvolite support, too? 

Louvolite does offer a Louvolite One Touch Home Hub, which is advertised to work with Alexa and Google Home, so I guess getting it to work with Homebridge should be in the realms of possibility. I can get my hands on a Louvolite hub to test it, if there is a chance it will work. Perhaps this plugin can already support it with some generic settings, or with minimal modifications? I don't know much about developing for Homebridge, but I am a software engineer, so can certainly tinker with it and try to get it to work (I may need some guidance though 😅).

**Comment by dubocr** [comment]
Hi,

I'm not certain to understand you need.
- You have Louvolite blind installed ?
- Do you already have a bridge to control them remotely (TaHoma for eg) ?

This plugin supports multiple platforms because the all share the same technical system (one IO-Homecontrol bridge developped by the same Somfy subsidiary).

If you blind are IO-Homecontrol and could be controled by an IO bridge, it should be compatible.

**Comment by navoshta** [comment]
Hi @dubocr — I appreciate you getting back to me, apologies if it sounded confusing: yep I do have Louvolite blinds installed, and I was on the fence whether I should get their hub or not, as I wasn't sure I would get it to work with Homebridge.

I've investigated a bit further by now, and, as you said, this plugin seems to work with blinds built on **Overkiz** platform. Unfortunately, Louvolite is built on top of another IoT platform called **Neo** (http://neosmartblinds.com)  — I managed to track down a couple of Homebridge packages that work with Neo, e.g. [matt-west/homebridge-neo-smart-blinds](https://github.com/matt-west/homebridge-neo-smart-blinds) and [jvmahon/homebridge-smartshades](https://github.com/jvmahon/homebridge-smartshades). They don't mention Louvolite brand explicitly, but I _think_ it should work, maybe with some tinkering. So I've ordered the official Louvolite hub and will give it a go.

I'll close this issue now, I'm leaving this info here in case someone else comes looking 🙂 — there is very little information on Louvolite blinds online.
